File: cpp/12a_59A_Word.cpp
#include<bits/stdc++.h>

int main(){
    std::string s;
    std::cin>>s;
    int upper = 0, lower = 0;
    for(auto i : s){
        if(isupper(i)){
            upper ++;
        }
        else{
            lower ++;
        }
    }
    if (upper > lower){
        for (auto &i : s){
            i = toupper(i);
        }
    }
    else{
        for(auto &i : s){
            i = tolower(i);
        }
    }
    std::cout<<s;
    return 0;
}

File: python/12a_59A_Word.py
def main():
    s = input()
    upper, lower = 0, 0
    ans = ""
    for i in s:
        if i.isupper():
            upper += 1
        else:
            lower += 1
    if upper > lower:
        for i in s:
            ans += i.upper()
    else:
        for i in s:
            ans += i.lower()
    print(ans)

if __name__ == "__main__":
    main()